### PR TITLE
Multiverse Sword Artefact/"There Can Be Only Me" admin verb (Ready to Merge)

### PR DESCRIPTION
**Pull request by ghost**
New item, the multiverse sword.

Deep lore: Summon clones of yourself from other dimension to fight by your side.

Gameplay: Activating the sword the first time will bind the sword to you. will summon a ghost as a "copy" of you. They will share your name, but have a randomized mutantrace, gender/appearance, and theme (clown, soviet, syndicate, etc). They will have their own copy of the sword (which shares a cooldown with all swords owned by people sharing your name). Their objective will depend on their the objectives of the original owner (either to take over the shuttle with clones, or to protect the innocent from the other mobs). The wizard clones will of course get the hijack objective.

The catch: If someone else, who is not the original owner or a clone, picks  up a sword, they can bind it to themselves and start summoning copies (on their own cooldown). When a non antagonist grabs the sword, there is a 30% chance to get the hijack objective, and a 70% chance to get the "protect the station" objective.

This will be in the wizards spell book as an artefact.

There will also be a new admin verb, "There Can Be Only Me," which gives everyone a sword, all access, and the hijack with clones objective. This should hopefully solve the respawn/gear problem in There Can Be Only One (though that verb will still exist).

Still have to do testing (I'm on a laptop which BYOND games crash) and tweaks.

### CODE REVIEW

**Comment by TheVekter** [comment]
CLONE ARMY TIME

**Comment by Cebasiz** [comment]
>there can be only one
>actually just kidding it summons clones

D R O P P E D

File: code/game/gamemodes/wizard/artefact.dm [hunk 1]
```diff
@@ -173,3 +173,254 @@
...
+					evil = FALSE
+		else
+			var/list/candidates = get_candidates(BE_WIZARD)
+			if(candidates.len)
+				var/client/C = pick(candidates)
+				spawn_copy(C, get_turf(user.loc), user)
+				user << "<span class='warning'>The sword flashes, and you find yourself face to face with...you!</span>"
+				charged = 0
+				spawn_copy(C, get_turf(user.loc), user)
+				for(var/obj/item/weapon/multisword/M in world)
```
Comment (Incoming5643) [inline code/game/gamemodes/wizard/artefact.dm]: I think there's a list for processing objects somewhere in the codebase, or you could define a global list specifically for these swords. Anything's better than a world call.

File: code/game/gamemodes/wizard/artefact.dm [hunk 2]
```diff
@@ -173,3 +173,254 @@
...
+					if(M.assigned == assigned)
+						M.charged = 0
+
+			else
+				user << "Unable to reach your apprentice! You can either attack the spellbook with the contract to refund your points, or wait and try again later."
```
Comment (Incoming5643) [inline code/game/gamemodes/wizard/artefact.dm]: :spaghetti:

**Comment by Incoming5643** [comment]
There's a practical problem in an item that can spawn antag status in that it can rapidly clog up a round end report. I took away the objectives from skeleton thralls for much the same reason. As long as you make it clear that clones are "on team you" when they're made it's still kosher for them to ACT like an antag even if mechanically they aren't.

**Comment by Dorsisdwarf** [comment]
No, just make it like the AI

"The survivors of Doctor Clonelord's army of minions wer played by [name],[name],[third nameand [fourth name]. However, [other ckey] [otherckey], and [thelastckey] were loyal minions, but unable to survive!"

**Comment by ike709** [comment]
Woo clone army... It's just like the time the geneticist turned everyone into identical copies of my character for a whole round.

**Comment by Iamgoofball** [comment]
do I get bonus points if I name myself Yuri and use this

**Comment by Kromgar** [comment]
Name yourself Barry Allen

**Comment by KorPhaeron** [comment]
Fixed what Incoming suggested, added the admin verb, added to spellbook, fixed various bugs, changed the icon to something flashier (energy katana). Ready to merge

@WJohn

**Comment by KorPhaeron** [comment]
Also @Cheridan @Miauw62  @Razharas @RemieRichards @Jordie0608

**Pull request merged**